tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -56,6 +56,14 @@ platform_tests/api/test_chassis.py::TestChassisApi::test_status_led:
 #######################################
 #####   api/test_chassis_fans.py  #####
 #######################################
+platform_tests/api/test_chassis_fans.py::TestChassisFans::test_get_fans_speed_tolerance:
+  #get_speed_tolerance API was disabled so platform code can perform fan tolerance checks 
+  #using RPM rather than thermalctld checking tolerance on percentages
```

Comment:
get_speed_tolerance API was disabled in order to allow for accurate fan tolerance calculations in platform code (instead of the inaccurate calculation upstream in thermalctld)
test_get_fans_speed_tolerance and test_set_fans_speed need to be skipped for all platforms, for both test_chassis and test_fan_drawer_fans until an upstream solution for fan tolerance is proposed and accepted.